spec/rspec/rails/matchers/have_http_status_spec.rb

```diff
@@ -7,6 +7,28 @@ def create_response(opts = {})
     }
   end
 
+  shared_examples_for "does not use deprecated methods for Rails 5.2+" do
+    it "does not use deprecated method for Rails >= 5.2" do
+      if 5 < Rails::VERSION::MAJOR ||
```

Comment:
Similarly might as well not define the test when not in the right rails version.